tests/core/test_oauth_clients.py

```diff
@@ -0,0 +1,58 @@
+from flask import json
+from newsroom.tests.users import (test_login_succeeds_for_admin, init as user_init)  # noqa
+from superdesk import get_resource_service
+import base64
+
+
+def test_oauth_clients(client):
+    test_login_succeeds_for_admin(client)
+    # Register a new client
+    response = client.post(
+        "/oauth_clients/new",
+        data=json.dumps({"name": "client1"}),
+        content_type="application/json",
+    )
+    assert response.status_code == 201
+
+    oauth_client = get_resource_service("oauth_clients").find_one(
+        req=None, name="client1"
+    )
+
+    # Check for the client secret
+    password = response.json.get("password", None)
+    if not password:
+        assert False
+
+    # OAuth Token Generation
+    username = response.json["_id"]
+    userpass = username + ":" + password
+    encoded_u = base64.b64encode(userpass.encode()).decode()
+
+    payload = {"grant_type": "client_credentials"}
+    token_auth_response = client.post(
+        "api/auth_server/token",
+        headers={"Authorization": "Basic %s" % encoded_u},
+        data=payload,
+    )
+
+    # Now use oauth token to access API
+    api_data_response = client.get(
```

Comment:
I wonder what are you testing here, we don't use the jwt token auth for the api, or do we?